packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue

```diff
@@ -224,20 +283,35 @@ interface BasicKnobs {
 	mapping: SimulateEnsembleMappingRow[];
 	weights: SimulateEnsembleWeight[];
 	numSamples: number;
-	timeSpan: TimeSpan;
+	method: CiemssMethodOptions;
+	endTime: number;
 }
 
 const knobs = ref<BasicKnobs>({
 	mapping: props.node.state.mapping,
 	weights: props.node.state.weights,
 	numSamples: props.node.state.numSamples,
-	timeSpan: props.node.state.timeSpan
+	method: props.node.state.method,
+	endTime: props.node.state.endTime
 });
 
+const activeAccordionIndicies = ref([0, 1, 2]);
```

Comment:
what are we planning to do that `activeAccordionIndicies` needs to be a ref? Or even a variable at all?

Reply:
I have found that when the Accordion component's :active-index is hardcoded to be [0,1,2] that when the user scrolls (or otherwise updates the page) it will reopen these tabs

Reply:
That might suggest that there is something weird with how the input pane is setup, because I am not sure why a state-update would trigger things to be completely re-rendered??

Reply:
Strange behaviour, but we decided to punt the investigation to a different issue.

https://github.com/DARPA-ASKEM/terarium/issues/5444